chain-impl-mockchain/src/ledger.rs

```diff
@@ -97,6 +97,9 @@ custom_error! {
         OldUtxoInvalidPublicKey { utxo: UtxoPointer, output: OutputOldAddress, witness: Witness } = "Old Transaction with invalid public key",
         AccountInvalidSignature { account: account::Identifier, witness: Witness } = "Account with invalid signature",
         MultisigInvalidSignature { multisig: multisig::Identifier, witness: Witness } = "Multisig with invalid signature",
+        TransactionHasTooManyInputs = "Transaction has more than 255 inputs",
+        TransactionHasTooManyOutputs = "Transaction has more than 255 outputs",
+        TransactionHasTooManyWitnesses = "Transaction has more than 255 witnesses",
```

Comment:
These messages should also use the constant definitions, or omit the numbers entirely.